components/textbox/textbox.tsx

```diff
@@ -234,6 +235,7 @@ export default class Textbox extends React.PureComponent<Props> {
                 </div>
             );
         }
+        const inputComponent = this.props.inputComponent || AutosizeTextarea;
```

Comment:
Should we configure this in `defaultProps` instead?

Reply:
Yes, good idea! And I took out the null as a possible value of `inputComponent` (`defaultProps` will not replace a null value).